Source/WebCore/css/parser/CSSParserImpl.h

```diff
@@ -125,7 +125,7 @@ class CSSParserImpl {
     bool consumeRuleList(CSSParserTokenRange, RuleListType, T callback);
 
     // This function updates the range it's given.
-    RefPtr<StyleRuleBase> consumeQualifiedRule(CSSParserTokenRange&, AllowedRulesType);
+    RefPtr<StyleRuleBase> consumeQualifiedRule(CSSParserTokenRange&, AllowedRulesType, StyleRule* parentRule);
```

Comment:
Would it make sense to have a default for parentRule?

`RefPtr<StyleRuleBase> consumeQualifiedRule(CSSParserTokenRange&, AllowedRulesType, StyleRule* parentRule = nullptr);`

Reply:
I don't have a strong opinion on this, but default value are fairly rare in the WebKit codebase though.